### PR TITLE
feat: Document the `class_code` field as returned by the /employer/employment endpoint.

### PR DESCRIPTION
**Pull request by aren55555**
Document the `class_code` field as returned by the /employer/employment endpoint.

I've been able to confirm this functionality in prod for:
* ADP Run ✅
* ADP WFN ✅
* Paycom ✅
* Gusto ✅
* Sandbox (WIP will add it to some of the individuals for `largeco`)
---------------------------------------
Won't do until requested by the developer:
* QBO (still in dev)
* Paychex (still in dev)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202482547603202

### CODE REVIEW

**Comment by aren55555** [comment]
For all adapters, the value will be null.

For adapters where we've explicitly added support (currently ADP Run, ADP WFN and Paycom) the value will be populated if and only if the provider had this data (aka they purchased their worker comp coverage thru the HRIS provider).

For Gusto, the WIP = work in progress, there's [an open PR](https://github.com/Finch-API/api-server/pull/2048) that @jason-gill00 is just adding unit testing to. Once this is merged and deployed Gusto will be the 4th provider where we will emit this class code (assuming the data was there).

**Comment by aren55555** [comment]
@tylernix try with an access token from the company named `LEAGUESIDE` (id = `d2938507-2c76-4601-ba9b-aa2676073a92`)... here is what I see:
Screen Shot 2022-06-21 at 3 55 08 PM

**Review by jason-gill00 — APPROVED** [review]
👍

File: reference/employer/openapi.yaml [hunk 1]
```diff
@@ -518,6 +519,10 @@ paths:
                               type: boolean
                               description: '`true` if the individual an an active employee or contractor at the company.'
                               nullable: true
+                            class_code:
+                              type: string
+                              description: "Worker's compensation'classification code for this employee"
```
Comment (seii) [inline reference/employer/openapi.yaml:524]: I'm guessing this should be "Worker's compensation classification..." (no stray apostrophe)?

Comment (aren55555) [reply reference/employer/openapi.yaml:524]: @seii see #73

**Pull request merged**